tests/atexit/enc/enc.cpp

```diff
@@ -51,7 +51,7 @@ void atexit_with_ecall_ecall(void)
 OE_SET_ENCLAVE_SGX(
     1,    /* ProductID */
     1,    /* SecurityVersion */
-    true, /* Debug */
-    8,    /* NumHeapPages */
-    8,    /* NumStackPages */
-    1);   /* NumTCS */
+    true, /* AllowDebug */
+    1024, /* HeapPageCount */
+    1024, /* StackPageCount */
+    1);   /* TCSCount */
```

Comment:
Revert the labels in comments.  They should match the labels that are recognized in .conf files so shouldn't change unless the .conf file parser is changed to recognize the new names.